README.md

```diff
@@ -43,32 +43,24 @@ $ docker run -d -it -p 8000:8000 -p 8001:8001 -p 8002:8002 \
 
 Wait for about a minute for MarkLogic to initialize before checking the ports.
 
-### Persistent Data Directory
+### Persistent Data Volume
 
-A MarkLogic Docker container is always instantiated with a volume (`/var/opt/MarkLogic`).
+A MarkLogic Docker container stores data in `/var/opt/MarkLogic` which should be persistent in Docker managed volume. It is reommended to use named volumes instead of bind mounts as per [Docker documentation](https://docs.docker.com/storage/volumes/).
 
-Run the following command to instantiate a container:
-
-```
-$ docker run -d -it -p 8000:8000 -p 8001:8001 -p 8002:8002 \
-     -e MARKLOGIC_INIT=true \
-     -e MARKLOGIC_ADMIN_USERNAME=<insert admin username> \
-     -e MARKLOGIC_ADMIN_PASSWORD=<insert admin password> \
-     PLACEHOLDER-FOR-DOCKER-IMAGE:DOCKER-TAG
-```
-
-Verify volume creation with this command:
+The following command will list previously created volumes:
 
 ```
 $ docker volume ls
 ```
 
-Alternately, you can bind a volume at runtime using the `-v` option:
+The command should output at least two randomly generated volume identifiers from the previous commands.
+
+The following command uses named volume in order to make it easier to manage:
```

Comment:
I think the following reads a little bit better but let me know what you think: 

`The following command uses a named volume in order to make management easier`